static/sass/styles.scss

```diff
@@ -835,3 +835,9 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
 .u-paper-bg {
   background-color: $color-paper;
 }
+
+
+.u-dark-code-snippet {
```

Comment:
You should make an issue on vanilla and link it here

Reply:
I'm not sure if we want this in Vanilla. @lyubomir-popov do we want to upstream the dark code snippet to Vanilla?